### PR TITLE
Replace dimagi email addresses with example addresses in localsettings example

### PR DESCRIPTION
**Pull request by NoahCarnahan**
@esoergel

### CODE REVIEW

File: localsettings.example.py [hunk 1]
```diff
@@ -87,22 +87,22 @@
...
-EXCHANGE_NOTIFICATION_RECIPIENTS = ['commcarehq-dev+exchange@dimagi.com']
+ADMINS = (('HQ Dev Team', 'commcarehq-dev+www-notifications@yourdomain.com'),)
+BUG_REPORT_RECIPIENTS = ['commcarehq-support@yourdomain.com']
+NEW_DOMAIN_RECIPIENTS = ['commcarehq-dev+newdomain@yourdomain.com']
+EXCHANGE_NOTIFICATION_RECIPIENTS = ['commcarehq-dev+exchange@yourdomain.com']
 
-SERVER_EMAIL = 'commcarehq-noreply@dimagi.com' #the physical server emailing - differentiate if needed
-DEFAULT_FROM_EMAIL = 'commcarehq-noreply@dimagi.com'
-SUPPORT_EMAIL = "commcarehq-support@dimagi.com"
+SERVER_EMAIL = 'commcarehq-noreply@yourdomain.com' #the physical server emailing - differentiate if needed
```
Comment (dimagimon) [inline localsettings.example.py]: E261 at least two spaces before inline comment
E262 inline comment should start with '# '

**Review by czue — APPROVED** [review]

**Comment by esoergel** [comment]
Note that whatever we put here will in all likelihood receive emails from some instance or another eventually.  If we put `None` or something (or comment out the variables), will the email backend fail in a way that the server admins would notice?  Alternatively, `example.com` is expressly meant for this [purpose](http://www.iana.org/domains/reserved)

**Comment by NoahCarnahan** [comment]
sweet, didn't know about example.com. That sounds like the right approach.

**Comment by NoahCarnahan** [comment]
Self merging since this has been approved

**Pull request merged**